src/main/java/ru/aston/service/MultitableServiceInterface.java

```diff
@@ -0,0 +1,12 @@
+package ru.aston.service;
+
+import java.util.List;
+
+public interface MultitableServiceInterface<T, S, E>{
+
+    List<S> getProject(E e);
+
+    List<T> getEmployeeOfPosition(E e);
+
+    List<T> getEmployeeOfProject(E e);
```

Comment:
The same problem that with repository. And should be `interface MultitableService`